### PR TITLE
Added install instructions for Arch Linux

### PR DESCRIPTION
**Pull request by naueramant**

### CODE REVIEW

**Comment by Yamboy1** [comment]
Can we change this to use the deno-bin package? I'm not a fan of the deno package as it requires building deno from source rather than just downloading the binary, and also the description of this repo says `Deno Binary Installer`

(DISCLAIMER: I am the maintainer for the deno-bin package)

File: README.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ brew install deno
 choco install deno
 ```
 
+**With [Yay (AUR)](https://aur.archlinux.org/packages/deno):**
+
+```sh
+yay -S deno
```
Comment (naueramant) [inline README.md:58]: Maybe just both @Yamboy1?
```suggestion
# From source
yay -S deno
# Pre-compiled
yay -S deno-bin
```

Comment (Yamboy1) [reply README.md:58]: Both works :smile:

**Comment by MarkTiedemann** [comment]
@naueramant @Yamboy1 What about "from source" & "pre-compiled"? Would you still like to add the pre-compiled `deno-bin`?

```diff
+ # From source
yay -S deno
+ # Pre-compiled
+ yay -S deno-bin
```

**Comment by MarkTiedemann** [comment]
This PR closes https://github.com/denoland/deno_install/issues/131.

**Comment by Yamboy1** [comment]
Yes, it would be nice to add both @MarkTiedemann

**Comment by naueramant** [comment]
Both also works for me, so sure let's merge the change with both.

**Comment by MarkTiedemann** [comment]
@Yamboy1 Since this PR is already merged, can you maybe create a follow-up PR?

**Comment by Yamboy1** [comment]
Yep sure

**Pull request merged**